### ISSUE TITLE
sabnzbd failing to build when media dir is >= 18 chars longs 

### ISSUE THREAD

**Issue by jamesmstone** [rasmus-kirk/nixarr#33]
**Describe the bug**

since (I believe) merging #24  I have been getting the following error:

```
/nix/store/bygsyckn582bhbvib0ssj5gvk46cpcwh-sabnzbd-set-user-values:13:80: E501 line too long (80 > 79 characters)
sab_config_map['misc']['download_dir'] = '/data/media/abcdef/usenet/.incomplete'
                                                                               ^
  copying dependency: /nix/store/8gb20n10vdx05gbfdnj220xpavnnsfpi-linux-6.6.41-modules/lib/modules/6.6.41/kernel/crypto/crct10dif_generic.ko.xz
  copying dependency: /nix/store/8gb20n10vdx05gbfdnj220xpavnnsfpi-linux-6.6.41-modules/lib/modules/6.6.41/kernel/lib/crc-t10dif.ko.xz
  dependency already copied: /nix/store/8gb20n10vdx05gbfdnj220xpavnnsfpi-linux-6.6.41-modules/lib/modules/6.6.41/kernel/lib/crc64.ko.xz
error: builder for '/nix/store/2xf2330z2sklk5ddsfjw88748g9ryz7a-sabnzbd-set-user-values.drv' failed with exit code 1;
       last 10 log lines:
       > sourcing setup hook '/nix/store/pag6l61paj1dc9sv15l7bm5c17xn5kyk-move-systemd-user-units.sh'
       > sourcing setup hook '/nix/store/jivxp510zxakaaic7qkrb7v1dd2rdbw9-multiple-outputs.sh'
       > sourcing setup hook '/nix/store/ilaf1w22bxi6jsi45alhmvvdgy4ly3zs-patch-shebangs.sh'
       > sourcing setup hook '/nix/store/cickvswrvann041nqxb0rxilc46svw1n-prune-libtool-files.sh'
       > sourcing setup hook '/nix/store/xyff06pkhki3qy1ls77w10s0v79c9il0-reproducible-builds.sh'
       > sourcing setup hook '/nix/store/ngg1cv31c8c7bcm2n8ww4g06nq7s4zhm-set-source-date-epoch-to-latest.sh'
       > sourcing setup hook '/nix/store/gps9qrh99j7g02840wv5x78ykmz30byp-strip.sh'
       > /nix/store/bygsyckn582bhbvib0ssj5gvk46cpcwh-sabnzbd-set-user-values:13:80: E501 line too long (80 > 79 characters)
       > sab_config_map['misc']['download_dir'] = '/data/media/abcdef/usenet/.incomplete'
       >                                                                                ^
       For full logs, run 'nix log /nix/store/2xf2330z2sklk5ddsfjw88748g9ryz7a-sabnzbd-set-user-values.drv'.
error: 1 dependencies of derivation '/nix/store/bjhrc6qrmybscac04mmm2yqi8d3f6i2m-unit-sabnzbd.service.drv' failed to build
error: 1 dependencies of derivation '/nix/store/xyzp1wfkmgcs1dhg71hv0fjyvw5j7vyw-system-units.drv' failed to build
error: 1 dependencies of derivation '/nix/store/b5xfybs50x66iqv0yp683zdqv2k03zxf-etc.drv' failed to build

```

I think it comes from this line https://github.com/rasmus-kirk/nixarr/pull/24/files#diff-a8006bdb367acb87ea9c25fe2b30ccb726677f930c6f110f40b6b82dc81102dbR24

which is then added to a python config file that is then checked with pylint. 

I am not sure of an easy solution but I imagine we could add an exclusion for this rule in this line 
https://github.com/rasmus-kirk/nixarr/pull/24/files#diff-a8006bdb367acb87ea9c25fe2b30ccb726677f930c6f110f40b6b82dc81102dbR59 

----

Thanks for the project. 

Aside: I think your website is down.

**Comment by jamesmstone** [comment]
note this still fails even when setting 

```
    sabnzbd.enable = false;
```

**Comment by rasmus-kirk** [comment]
Thanks for reporting this, I'll have a look sometime. Also, the website has recently been changed to https://nixarr.com/ are you using that one?

@rlad78

**Comment by jamesmstone** [comment]
Thanks!

That's the url, certificate issue for me


________________________________

**Comment by rasmus-kirk** [comment]
Pushed a hotfix to the dev branch, can you test that it now works if `sabnzbd.enable = false;`?

**Comment by jamesmstone** [comment]
Sorry I can't see the hotfix. Do I need to look at a particular branch?

Website is looking good now.

Tusind tak!




________________________________

**Comment by rlad78** [comment]
> Pushed a hotfix to the dev branch, can you test that it now works if `sabnzbd.enable = false;`?

Works for me with `sabnzbd.enable = false`. Looks like we need `pkgs` imported though in `nixarr/sabnzbd/default.nix`:
```nix
       error: undefined variable 'pkgs'

       at /nix/store/yg0nqdv70p6l09fbm73rspc1a8gslqrz-source/nixarr/sabnzbd/default.nix:125:37:

          124|
          125|     fix-config-permissions-script = pkgs.writeShellApplication {
             |                                     ^
          126|       name = "sabnzbd-fix-config-permissions";
```

The error is due to [the pep8 linter being on by default](https://github.com/NixOS/nixpkgs/issues/269811). I'll submit a fix in a bit.

**Comment by rasmus-kirk** [comment]
Should be solved in #34
